spec.bs

```diff
@@ -604,6 +614,12 @@ authentication:
     payment instrument {{PaymentCredentialInstrument/displayName}} and
     {{PaymentCredentialInstrument/icon}}.
 
+    NOTE: If the icon could not be fetched or decoded, then
+    {{PaymentCredentialInstrument/iconMustBeShown}} must be `false` here as
+    otherwise the [[#sctn-steps-to-check-if-a-payment-can-be-made|the steps to check if a payment can be made]]
+    would have failed. In this case, the User Agent may show no icon or a
+    generic payment instrument icon in its place.
+
```

Comment:
Apologies for being *really* picky, but I think we should keep the 'the User Agent may show no icon or a generic payment instrument icon in its place' as part of the normative text above, and let the note explain why that could happen. That is, something like:

```suggestion
    payment instrument {{PaymentCredentialInstrument/displayName}} and
    {{PaymentCredentialInstrument/icon}}. If an image resource could not be
    fetched or decoded from the input {{PaymentCredentialInstrument/icon}},
    then the User Agent may show no icon or a generic payment instrument
    icon in its place.

    NOTE: If the icon could not be fetched or decoded, then
    {{PaymentCredentialInstrument/iconMustBeShown}} must be `false` here as
    otherwise the [[#sctn-steps-to-check-if-a-payment-can-be-made|the steps
    to check if a payment can be made]] would have failed previously.
```

Reply:
Done